inc/thorvg.h

```diff
@@ -1848,6 +1848,15 @@ class TVG_API Animation
      * @note Experimental API
      */
     float totalFrame() const noexcept;
+    
+    /**
+     * @brief Retrieves the start frame number specified by segment.
+     *
+     * @return Start frame number specified by segment. If there is no segment returns first frame;
+     *
+     * @note Experimental API
+     */
+    float startFrame() const noexcept;
```

Comment:
I'd like to say, the startFrame() API is not necessary. Once a segment is established, the frame number could be internally adjusted by the engine (lottie loader), eliminating the need for users to handle this directly.

Here's an overview of related APIs:
```
float Animation::totalFrame()  //Returns the total number of frames in the segment.
float Animation::duration()     //Returns the duration of the segment.
float Animation::curFrame()   //Returns the current frame number, remapped to the segment (i.e., the current frame number - the segment's starting frame number).
float Animation::frame(float no)  //Remaps the frame number (no) to the segment(i.e., @p no + the segment's starting frame number).
```